usr/share/rear/restore/VEEAM/default/500_restore.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Agent registration and restore command
+LogPrint "Deleting Veeam SQLite Linux Agent Database"
+rm -rf /var/lib/veeam/*
+
+LogPrint "Starting veeamservice agent for linux"
+systemctl start veeamservice
+StopIfError "Failed to start veeamservice Agent for Linux"
+sleep 5
+
+# Debug stuff
+#systemctl status veeamservice
+#veeamconfig mode info <if mode: Job, then run mode reset>
+
+LogPrint "Accept Veeam EULA agreement"
+# create directories for EULA agreement
+mkdir -p /usr/share/doc/veeam/
+touch /usr/share/doc/veeam/EULA
+touch /usr/share/doc/veeam/3rdPartyNotices.txt
+
+echo yes > /tmp/answer.txt
+echo yes >> /tmp/answer.txt
+
+LogPrint "Query available Veeam VBR Server"
+cat /tmp/answer.txt | veeamconfig vbrServer list
+StopIfError "Failed to agree Veeam EULA"
+sleep 2
+
+LogPrint "Reset Veeam Client configuration mode from policy to standalone"
+veeamconfig mode reset
+#StopIfError "Failed to reset client configuration mode to standalone"
+sleep 5
+
+LogPrint "Registering Veeam Linux Client Agent to Veeam VBR backup server"
+veeamconfig vbrServer add --name $VBRSRV --address $VBRADDR --domain $VBRDOMAIN --login $VBRUSR --password $VBRPASSWD
+StopIfError "Failed to register Veeam Linux Client Agent to Veeam VBRServer $VBRSRV"
+
+sleep 20
+#
+LogPrint "Query the latest available full backup for the Linux Agent and mount it to /mnt/backup"
+for i in `veeamconfig backup list --all | awk '{ print $4 }' | tail -1` ; do echo $i ; veeamconfig backup mount --id $i ; echo veeammount -d /tmp/veeamflr/$i/FileLevelBackup_0 -p /mnt/backup -o ro -m ; veeammount -d `ls -ltrhd /tmp/veeamflr/* | awk '{ print $9 }' | awk '{
```

Comment:
Don't use backticks, see
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-read

Why almost the whole 'for' loop as a single long line of commands?